src/org/parosproxy/paros/extension/option/OptionsViewPanel.java

```diff
@@ -77,6 +82,8 @@
 	private static final String TIME_STAMP_FORMAT_TIMEONLY =  Constant.messages.getString("timestamp.format.timeonly");
 	
 	private JPanel panelMisc = null;
+	private JScrollPane viewsScrollPane;

```

Comment:
`views` should be singular (but `mainScrollPane` or just `scrollPane` would be fine too).